packages/offix-datastore/src/replication/GraphQLReplicator.ts

```diff
@@ -33,35 +34,55 @@ export interface ReplicatorMutations {
  * GraphQL mutations for create, update and delete
  */
 export interface ReplicatorSubscriptions {
-  /**
-   * GraphQL create subscription document.
-   */
-  new: string | DocumentNode;
-
-  /**
-   * GraphQL update subscription document.
-   */
-  updated: string | DocumentNode;
-
-  /**
-   * GraphQL delete subscription document.
-   */
-  deleted: string | DocumentNode;
+    /**
+     * GraphQL create subscription document.
+     */
+    new: string | DocumentNode;
```

Comment:
actually `new` is a keyword so we might avoid this name :) 
My bad!

Reply:
Won't be a problem.